src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable-next-line import-notation */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
 body {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
```

Comment:
add this style to reset.css

Reply:
ok

Reply:
`list-style: none;` should go to reset, as @mrkovaliv noted